GitExtensionsVSIX/source.extension.vsixmanifest

```diff
@@ -24,4 +26,7 @@
     <Assets>
         <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
     </Assets>
+    <Prerequisites>
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[14.0,16.0)" DisplayName="Visual Studio core editor" />
```

Comment:
- Visual Studio 11.0 is VS2012
- Visual Studio 12.0 is VS2013
- Visual Studio 14.0 is VS2015.

I am not certain what the correct version range should be to be honest. I think VSIX is mandatory from VS2013 (which is 12.0) so I used 12.0 as minimum version.